tests/MoBi.Tests/Presentation/ProjectTaskSpecs.cs

```diff
@@ -110,6 +121,45 @@ public void should_update_favorites_in_projects()
       {
          _simulationTransfer.Favorites.Each(favorite => _project.Favorites.ShouldContain(favorite));
       }
+
+      [Observation]
+      public void should_update_default_simulation_settings_in_project()
+      {
+         A.CallTo(() => _interactionTasksForSimulationSettings.UpdateDefaultSimulationSettingsInProject(_simulationTransfer.Simulation.Settings.OutputSchema, _simulationTransfer.Simulation.Settings.Solver)).MustNotHaveHappened();
+      }
+
+      [Observation]
+      public void should_update_default_output_selections_in_project()
```

Comment:
naming is off here

Reply:
Sorry, don`t get it. How should this be named then?

Reply:
The names indicate that the simulation settings should be updated, but the tests are checking that the settings are not updated.

Reply:
oh I See... sorry I`m dumb!